spec.html

```diff
@@ -29879,7 +29880,9 @@
           1. Let _len_ be ? ToLength(? Get(_O_, `"length"`)).
           1. If _len_ is 0, return -1.
           1. If argument _fromIndex_ was passed, let _n_ be ? ToInteger(_fromIndex_); else let _n_ be _len_-1.
-          1. If _n_ &ge; 0, let _k_ be min(_n_, _len_ - 1).
+          1. If _n_ &ge; 0, then
+            1. Let _k_ be min(_n_, _len_ - 1).
+            1. NOTE: Applying the mathematical function min(&hellip;) converts *-0* to *+0*.
```

Comment:
similar question here - perhaps an explicit step to handle `-0` rather than a note?